src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -762,8 +762,13 @@ public final ConveyorTile dispatch(
         }
 
         if (!tld.layerExists(layerName)) {
-            requestMistmatchTarget.append("not a tile layer");
-            return null;
+            // GEOS-9431-check in catalog also if not found TileLayerDispatcher
+            LayerInfo layerInfo = catalog.getLayerByName(layerName);
+            if (layerInfo != null) layerName = layerInfo.prefixedName();
```

Comment:
This breaks the workflow of GWC, the check should be done by ``!tld.layerExists(prefixedName)``.

Reply:
understood, will do